test/controllers/api/v2/report_templates_controller_test.rb

```diff
@@ -379,7 +379,7 @@ def expect_job_enque_with(input_values, mail_to: nil, delay_to: nil, format: nil
     end
 
     def stub_plan(opts = {})
-      opts.keys.each { |k| plan.send("#{k}=", opts[k]) }
+      opts.each_key { |k| plan.send("#{k}=", opts[k]) }
```

Comment:
It would be better to iterate both keys and values to avoid a lookup (even if it's only test code):
```suggestion
      opts.each { |k, v| plan.send("#{k}=", v) }
```